modules/admin_manual/pages/release_notes.adoc

```diff
@@ -560,6 +560,44 @@ Version 10.1 delivers all the prerequisites to be compatible with the Microsoft
 
 ownCloud Server 10.1 introduces WebDAV Locks that allow clients to lock and unlock resources to prevent other users from making changes. The feature has been implemented as a prerequisite for manual file locking and MS Office Online Server compatibility. In the current state, file locking is only available via API. Users can recognize locked files via the "lock" icon in the file list. Additionally a lock owner (the user who locked the file) can manually unlock them via the "Locks" tab in the right sidebar. The "Locks" tab will only appear for files that have active locks.
 
+=== Foreign Keys in Database
+
+Please note that foreign keys have been added with the :xref:webdav-locks[WebDAV Locks] feature.
+This is the first time ownCloud implements foreign keys.
+
+==== How Does This Affect Each Database
+
+===== SQLite
+
+Foreign keys are, by default, disabled in SQLite.
+Here is what https://sqlite.org/foreignkeys.html[the current documentation] says about enabling foreign key support:
+
+[quote,Enabling Foreign Key Support]
+To enable foreign key support, the library must be compiled with neither `SQLITE_OMIT_FOREIGN_KEY` or `SQLITE_OMIT_TRIGGER` defined.
+If `SQLITE_OMIT_TRIGGER` is defined but `SQLITE_OMIT_FOREIGN_KEY` is not, then SQLite behaves as it did prior to version 3.6.19 (2009-10-14) - foreign key definitions are parsed and may be queried using PRAGMA foreign_key_list, but foreign key constraints are not enforced.
+The PRAGMA foreign_keys command is a no-op in this configuration.
+If `OMIT_FOREIGN_KEY` is defined, then foreign key definitions cannot even be parsed (attempting to specify a foreign key definition is a syntax error).
+
+===== MySQL
+
+MySQL supports foreign keys. 
+They are not disabled by default.
```

Comment:
```suggestion
They are enabled by default.
```
Write a positive sentence rather than the sort-of-double-negative?
(and in 3 more place below)